packages/frontend/mobile/src/router.tsx

```diff
@@ -13,6 +16,25 @@ import { Component as Home } from './pages/workspace/home';
 import { Component as Search } from './pages/workspace/search';
 import { Component as Tag } from './pages/workspace/tag';
 import { Component as TagDetail } from './pages/workspace/tag/detail';
+import { AllWorkspaceModals } from './provider/model-provider';
+
+function RootRouter() {
+  const navigate = useNavigate();
+  const [ready, setReady] = useState(false);
+  useEffect(() => {
+    // a hack to make sure router is ready
+    setReady(true);
```

Comment:
any detail?

Reply:
Not sure ... this is copied from '@affine/core/router';

Reply:
approve but need to be refactored